frontend/assets/translations/fr.json

```diff
@@ -1,921 +1,918 @@
+    },
+    "Calendar": {
+      "displayName": "Calendrier",
+      "description": "Calendrier"
+    },
+    "Iframe": {
+      "displayName": "Iframe",
+      "description": "Afficher un iframe"
+    },
+    "CodeEditor": {
+      "displayName": "Éditeur de code",
+      "description": "Éditeur de code"
+    },
+    "Tabs": {
+      "displayName": "Onglets",
+      "description": "Composant des onglets"
+    },
+    "Timer": {
+      "displayName": "Minuteur",
+      "description": "minuteur"
+    },
+    "Listview": {
+      "displayName": "Affichage de la liste",
+      "description": "Wrapper pour plusieurs composants"
+    },
+    "Tags": {
+      "displayName": "Mots clés",
+      "description": "Le contenu peut être affiché sous forme de balises"
+    },
+    "Pagination": {
+      "displayName": "Pagination",
+      "description": "Pagination"
+    },
+    "CircularProgressbar": {
+      "displayName": "Barre de progrès circulaire",
+      "description": "Montrer les progrès en utilisant la barre de progression circulaire"
+    },
+    "Spinner": {
+      "displayName": "Fileuse",
+      "description": "Le spinner peut être utilisé pour afficher l'état de chargement"
+    },
+    "Statistics": {
+      "displayName": "Statistiques",
+      "description": "Les statistiques peuvent être utilisées pour afficher différentes informations statistiques"
+    },
+    "RangeSlider": {
+      "displayName": "Curseur de gamme",
+      "description": "Peut être utilisé pour montrer le curseur avec une gamme"
+    },
+    "Timeline": {
+      "displayName": "Chronologie",
+      "description": "Représentation visuelle d'une séquence d'événements"
+    },
+    "SvgImage": {
+      "displayName": "Image SVG",
+      "description": "Image SVG"
+    },
+    "Html": {
+      "displayName": "Visionneuse HTML",
+      "description": "Visionneuse HTML"
+    },
+    "VerticalDivider": {
+      "displayName": "Diviseur vertical",
+      "description": "Séparateur vertical entre les composants"
+    },
+    "CustomComponent": {
+      "displayName": "Composant personnalisé",
+      "description": "Ajoutez votre composant React personnalisé"
+    },
+    "ButtonGroup": {
+      "displayName": "Groupe de boutons",
+      "description": "Coup de pied"
+    },
+    "PDF": {
+      "displayName": "Pdf",
+      "description": "Intégrer le fichier pdf"
+    },
+    "Steps": {
+      "displayName": "Pas",
+      "description": "Pas"
+    },
+    "KanbanBoard": {
+      "displayName": "Conseil d'administration de Kanban",
+      "description": "Conseil d'administration de Kanban"
+    },
+    "ColorPicker": {
+      "displayName": "Pipette à couleurs",
+      "description": "Palette de séjour de couleur"
+    },
+    "TreeSelect": {
+      "displayName": "Sélection de l'arborescence",
+      "description": "Sélectionnez des valeurs dans une vue d'arbre"
+    }
+  },
+  "leftSidebar": {
+    "Inspector": {
+      "text": "Inspectrice",
+      "tip": "Inspectrice"
```

Comment:
```suggestion
      "text": "Inspecteur",
      "tip": "Inspecteur"
```

Reply:
@roiLeo, did the changes are you instructed, please review for any mistakes.